scopes.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Access scopes
+---
+
+The diaspora* API features access scopes (or just 'scopes') in order to implement resource access management for API clients.
+
+Scopes are part of [OAuth 2.0 specification][oauth2] which is a base for [OpenID Connect Core 1.0][connect] implemented by diaspora*.
+
+When the API client requests an access token it sends the list of scopes it wants to get access to according to the selected authorization flow (see ["Authentication"][authentication]).
+
+Granted scopes may differ from what was requested by the client.
```

Comment:
FYI in the current UI this is an all or nothing thing.  It lists the permissions that the app is requesting when it tries to get an access token but there are no checkboxes.  I don't know if we want the fine grained controls to be in MVP1 or a small update afterward.

Reply:
OpenID Connect allows opt out of some of the requested scopes and I think it is fine to have it in our UI, but IMO for MVP1 it is okay the way it is.